bindings/rust/Cargo.toml

```diff
@@ -21,6 +21,9 @@ glob = "0.3.1"
 rand = "0.8.5"
 serde_yaml = "0.9.17"
 
+[build-dependencies]
+bindgen = { git = "https://github.com/rust-lang/rust-bindgen" , rev = "0de11f0a521611ac8738b7b01d19dddaf3899e66" }
```

Comment:
Any specific features we are using from this particular commit?

Reply:
but this should be release soon